### ISSUE TITLE
Move System Container to WEB-INF/velocity

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#21853]
A follow on to #21265.  Just like we did with the system template, we need to move the system containers to WEB-INF/velocity so they are automatically included in new AND upgraded dotCMS installations.

**Comment by jcastro-dotcms** [comment]
Hey @fmontes ! FYI, the following REST Endpoint:
```
/api/v1/containers?per_page=5&orderby=title&page=1&direction=ASC&per_page=5
```
Needs to pass down an additional `&system=true` parameter so that the Container API returns the System Container.

**Comment by fmontes** [comment]
We can fix the FE now.

**Comment by bryanboza** [comment]
We need some work here,  we need be able to use the system container when you are creating a new template from template designer or advanced template, right now the call to list the containers is not including the system container. 
![Screen Shot 2022-05-27 at 12 20 03 PM](https://user-images.githubusercontent.com/2641437/170787032-eacedc12-73f6-406a-b2cc-cfe505c6a8da.png)

**Comment by bryanboza** [comment]
We need some work when you add the system container to an advanced template 

https://user-images.githubusercontent.com/2641437/173459115-8457f7de-be57-4d22-a6e3-c853ce83250f.mov

**Comment by bryanboza** [comment]
Fixed, tested on release-22.06 // Docker // FF
